README.md

```diff
@@ -52,15 +47,29 @@ Install from the Eclipse marketplace.
 
 
 ## Repository
-Download the repo in a zip file. 
+The Eclipse repositories for this plugin.
+
+### Production
+Release update site.
+
+* [Eclipse Marketplace entry for the GWT Plugin](https://marketplace.eclipse.org/content/gwt-plugin)
+* [Update sites for the GWT Plugin](https://plugins.gwtproject.org/eclipse/gwt-eclipse-plugin/)
 
-* [https://github.com/gwt-plugins/gwt-eclipse-plugin/releases/download/v4.0.0/repository.zip](https://github.com/gwt-plugins/gwt-eclipse-plugin/releases/download/v4.0.0/repository.zip)
+### Production Zip
+Download the repo in a zip file from the latest release at the
+[releases page](https://github.com/gwt-plugins/gwt-eclipse-plugin/releases/). The current release is
+[4.0.0](https://github.com/gwt-plugins/gwt-eclipse-plugin/releases/download/v4.0.0/repository.zip)
 
+### Staging
+The staging repository is also at plugins.gwtproject.org, under the `nightly` version. At this time,
+these are signed with a self-signed certificate.
 
-## Development 
+* Nightly update site: https://plugins.gwtproject.org/eclipse/gwt-eclipse-plugin/nightly
```

Comment:
Any plan for uploading nightly artifacts as part of the build?

Reply:
Now that I think we've got the build right, nightly artifacts are next, I'll be trying to implement a webhook for this.